### PR TITLE
Document running under WSGI server.

### LINKED ISSUE #30: WSGI deployment documentation
Recently Zope WSGI publishing seems to get some love. Is there any documentation around how to setup a Zope with WSGI? Are there any best practises or example configuration?

### PR DESCRIPTION
**Pull request by tseaver**
- Add docs for installation into a virtualenv.
- Update Sphinx paths to use `_` prefix.

Closes #30.

### CODE REVIEW

**Comment by sheilatron** [comment]
This is very helpful! Thank you @tseaver !!!

File: doc/INSTALL-virtualenv.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+.. code-block:: sh
+
+  $ bin/mkzopeinstance
+
+You can specify the Python interpreter to use for the instance
+explicitly:
+
+.. code-block:: sh
+
+  $ bin/mkzopeinstance --python=bin/pythn
```
Comment (tomgross) [inline doc/INSTALL-virtualenv.rst]: it should read  $ bin/mkzopeinstance --python=bin/python

**Comment by tomgross** [comment]
:+1: Works perfectly. Thanks @tseaver !

File: doc/WSGI.rst [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+   [app:zope-app]
+   use = egg:Zope2#main
+   zope_conf = %(here)s/zope.conf
+
+   [pipeline:zope-pipeline]
+   pipeline =
+       egg:paste#evalerror
+       egg:repoze.retry#retry
+       egg:repoze.tm2#tm
+       zope
```
Comment (tomgross) [inline doc/WSGI.rst]: This should be **zope-app** in this example.

**Comment by aclark4life** [comment]
Nice :beers:

**Pull request merged**